src/common/components/ExternalLink.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { Link } from '@mui/material';
+
+import { useAnalytics } from 'monitoring/analytics';
+
+const ExternalLink = ({ href, children }) => {
```

Comment:
Add a comment explaining why this component exists and linking to the Plausible GitHub issue.